src/script/run_tox.sh

```diff
@@ -91,17 +95,7 @@ function main() {
     # tox.ini will take care of this.
     export CEPH_BUILD_DIR=$build_dir
 
-    if [ "$with_python2" = "ON" ]; then
-        ENV_LIST+="py27,"
-    fi
-    # WITH_PYTHON3 might be set to "ON" or to the python3 RPM version number
-    # prevailing on the system - e.g. "3", "36"
-    if [[ "$with_python3" =~ (^3|^ON) ]]; then
-        ENV_LIST+="py3,"
-    fi
-    # use bash string manipulation to strip off any trailing comma
-    ENV_LIST=${ENV_LIST%,}
-    tox -c $tox_path/tox.ini -e "${ENV_LIST}" "$@"
+    tox -c $tox_path/tox.ini -e "$tox_envs" "$@"
```

Comment:
Minor thing. What about:
```suggestion
    tox -c $tox_path -e "$tox_envs" "$@"
```
`tox` auto-searches for `tox.ini` in the specified dir. By leaving, `tox_path` we may allow components to specify have multiple tox INI files for different test stages: `tox-unit.ini`, `tox-integration.ini`, ....

Reply:
@epuertat i will read tox's document and include your suggestion in a follow-up PR.